src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -162,6 +162,63 @@ describe('manage nsfs cli bucket flow', () => {
             expect(path_exists).toBe(false);
         });
     });
+
+    describe('cli update bucket', () => {
```

Comment:
In general, we have valid cases for bucket update in the file "test_nc_nsfs_cli" (mocha), for example:
- 'cli bucket update owner'
- 'cli bucket update bucket policy'
- 'cli bucket update bucket policy - delete bucket policy'
- 'cli bucket update bucket name'
I created this `describe` because it is preferred to add new tests in Jest.